File: frontend/src/features/actividades/rectificatoria_legal_carta_resguardo/models/rectificatoria_legal_carta_resguardo.ts
export interface RectificatoriaLegalCartaResguardo {
  id_rectificatoria_legal_carta_resguardo: number;
  id_expediente: number;
  id_usuario_solicitante: number;
  observaciones?: string | null;
  is_active: boolean;
  row_status: boolean;
  created_by: number;
  created_date: string;
  modified_by?: number | null;
  modified_date?: string | null;
}

File: backend/Multibanca.Domain.Models/Multibanca/rectificatoria_legal_carta_resguardo.cs
using Common.Domain.Models;

namespace Multibanca.Domain.Models.Multibanca
{
    public class rectificatoria_legal_carta_resguardo : base_auditoria
    {
        public int id_rectificatoria_legal_carta_resguardo { get; set; }
        public long id_expediente { get; set; }
        public int id_usuario_solicitante { get; set; }
        public string? observaciones { get; set; }
    }
}
